### PR TITLE
feat: added basics for upload

### PR DESCRIPTION
**Pull request by jjmachan**

### CODE REVIEW

File: src/ragas/utils.py [hunk 1]
```diff
@@ -22,6 +25,9 @@
 }
 # endpoint for uploading results
 RAGAS_API_URL = "https://api.ragas.io"
+RAGAS_APP_URL = "https://app.ragas.io"
+RAGAS_API_VERSION = "1.0.0"
```
Comment (ganeshrvel) [inline src/ragas/utils.py:29]: RAGAS_API_VERSION = "1.0.0"

app api version should be picked up from the internals. Not via env

File: src/ragas/utils.py [hunk 2]
```diff
@@ -22,6 +25,9 @@
...
+RAGAS_API_SOURCE = "ragas_py"
```
Comment (ganeshrvel) [inline src/ragas/utils.py:30]: RAGAS_API_SOURCE should be hardcoded. Not picked up via env

**Review by ganeshrvel — CHANGES_REQUESTED** [review]

File: src/ragas/utils.py (hunk 2 above)
Comment (jjmachan) [reply src/ragas/utils.py:29]: it is hard coded - its a constant

**Review by shahules786 — APPROVED** [review]
LGTM

**Pull request merged**